calculator.js

```diff
@@ -0,0 +1,34 @@
+var calculator = {
+  memory : 0,
+  clearMemory: function () {
+    memory=0;
```

Comment:
either use `calculator.memory` or better, `this.memory` to update that attribute. currently `memory` is a global variabel